Assets/Scripts/trial/FieldTrial.cs

```diff
@@ -19,13 +20,26 @@ public FieldTrial(InputField[] fields) : base(-1, -1)
         {
             _fields = fields;
 
-            // This block of code basically calls the directory picker to select the configuration file.
-            while (true)
+            // This block of code will use the default configuration file if
+            // using webGL or Android (can add others in if statement) 
+            // Otherwise calls the directory picker to select the configuration file.
+
+            string[] files = Directory.GetFiles(Application.streamingAssetsPath + "/Default_Config/");
+            Debug.Log(files);
```

Comment:
Consider leaving a more meaningful Debug.Log statement rather than just Debug.Log(files).

Reply:
Removed this - I was just using it to make sure i was getting the correct info.